docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_label: "Installation"
+title: ""
+---
+
+# Airgap installation
+
+This guide will show you how to install Kubewarden in air-gapped environments. In an air-gapped installation of Kubewarden, 
+you will need a private OCI registry that is located somewhere accessible by your kubernetes cluster. Kubewarden Policies 
+are WebAssembly modules, therefore they can be stored inside of an OCI compliant registry as OCI artifacts.
+You need to add Kubewarden's images and policies to this OCI registry. Let's see how to do that.
```

Comment:
I would like to have a "Requirements" section where we list all the things a user must have:

* A registry to store container images **and** policies
* A list of on-premise registries that we know can support OCI artifacts
* kwctl version X
* docker version X
* anything I might have forgot

I think it would be nice to have that as a dedicated page like:
```
Operator manual
|
 `- Air gap
    |
     `- Requirements
     `- Setup
```

Reply:
done https://github.com/kubewarden/docs/pull/150/commits/63a8525fc4363b5abb564515310a6941886e6d76